.gitignore

```diff
@@ -79,6 +80,7 @@ out/
 
 # Temporary files
 *.~vsdx
+..gitignore.swp
```

Comment:
??

Reply:
I am going to create a new PR with this line changed to `*.swp`